src/node_crypto.cc

```diff
@@ -3320,6 +3320,7 @@ void Hmac::HmacDigest(const FunctionCallbackInfo<Value>& args) {
 
   bool r = hmac->HmacDigest(&md_value, &md_len);
   if (!r) {
+    delete[] md_value;
```

Comment:
This isn't correct and is a false positive. `HmacDigest()` only [returns false immediately](https://github.com/nodejs/node/blob/870229e66529309dfea932c52d718ddc2d734966/src/node_crypto.cc#L3296-L3297), so `md_value` is never touched. So this will effectively try to delete a null pointer.